### PR TITLE
Fix page scroll

### PR DESCRIPTION
**Pull request by simoneduca**
Move background element closing tag. Add border-radius to logo.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,205 +1,163 @@
...
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <!-- Add jQuery library -->
+    <script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
```
Comment (rogerhutchings) [inline public/index.html:12]: Just out of interest, what do you need jQuery for?

Comment (rogerhutchings) [inline public/index.html:12]: Oh right, fancybox

File: public/style.css [hunk 2]
```diff
@@ -1,8 +1,5 @@
 @import url('http://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700,700italic');
 @import url('http://fonts.googleapis.com/css?family=Montserrat&text=ZOONIVERSE');
```
Comment (rogerhutchings) [inline public/style.css]: Had a quick look, can't see where this font is being used

Comment (simoneduca) [inline public/style.css]: You're right. I just grabbed it from the live project. They might be using it somewhere else. I'll remove it.

**Pull request merged**